app/main/views/g_cloud.py

```diff
@@ -209,7 +209,7 @@ def get_service_by_id(service_id):
         status_code = 200
         if service['serviceMadeUnavailableAuditEvent'] is not None:
             service_unavailability_information = {
-                'date': dateformat(service['serviceMadeUnavailableAuditEvent']['createdAt']),
+                'date': dateformat(framework['frameworkExpiresAtUTC']),
```

Comment:
I've just realised we're using `service_unavailability_information.date` in two places on the template. The first case is the bug we're trying to solve, where the framework has expired. The second case is where the supplier has disabled the service themselves (which can happen at any time during the framework). So we do need to provide `(service['serviceMadeUnavailableAuditEvent']['createdAt']` to the template context.

Reply:
@katstevens Ahh, OK, makes sense.

May I ask in the case that the framework has expired do we also want to return 410 as we do if the service is unavailable? i.e. in: https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/baad6dd06a5ef1ce54c06cd5f03f2d3c8091b73c/app/main/views/g_cloud.py#L215-L216

Reply:
In theory yes. However that might be a separate tech debt ticket 😸